api/envoy/config/core/v3/protocol.proto

```diff
@@ -149,6 +166,12 @@ message AlternateProtocolsCacheOptions {
   // alternate protocols entries to disk.
   // This function is currently only supported if concurrency is 1
   TypedExtensionConfig key_value_store_config = 3;
+
+  // Allows pre-populating the cache with entries, to reduce odds of using TCP for the
+  // first request to endpoints expected to support HTTP/3.
+  // Note that if a given entry is present in the key value store, and pre-populated_entries, the key
+  // value store value will be used.
```

Comment:
I was expecting to see this bit of documentation above while I was reading it, maybe it should be in both places?